pkg/api/helper/helpers.go

```diff
@@ -594,3 +594,27 @@ func PersistentVolumeClaimHasClass(claim *api.PersistentVolumeClaim) bool {
 
 	return false
 }
+
+// GetTopologyConstraintsFromAnnotations gets the json serialized topology constraints data from PersistentVolume.Annotations
+// and converts it to the []TopologyConstraint type in api.
+// TODO: remove when alpha support for topology constraints is removed
```

Comment:
We don't have to.  I'm just wondering based off of storageclasses, where we are still "supporting" the alpha annotation even though it's now GA.